mlx/version.h

```diff
@@ -0,0 +1,24 @@
+// Copyright © 2023-2024 Apple Inc.
```

Comment:
I think it's worth including this in `mlx.h` by default. It's low footprint and probably expected to be there if you do `"#include <mlx.h>`.

Reply:
Fixed